### ISSUE TITLE
'Test Cases' table is not supported inside resource file

### ISSUE THREAD

**Issue by nefethael** [nokia/RED#339]
Hi team,

I'm encountering `'Test Cases' table is not supported inside resource file` error for every `.robot` testsuites. I'm using tab separator format and Red v0.9.0:

Example test_write_coil.robot:

```
| *** Setting ***  
| Documentation   | Test modbus Write COIL
| Resource        | ../resources/modbus.resource

| *** Test Case ***  
| Test Write Coil
| | Write Coil | 0 | 0
```

Is it a known problem?
Best regards,
Vincent

**Comment by michalanglart** [comment]
No, similarly as in #338 - it seems our users very rarely use pipe separators, so no one reported it so far. Looks easy too, so probably we'll fix that in 0.9.1

**Comment by nefethael** [comment]
@michalanglart Thanks, removing trailing spaces fixes the issue.

**Comment by michalanglart** [comment]
Corrected in 0.9.1
